Can't run `yarn` in project due to `node-gyp` failures

This morning I attempted to post a PR that upgrades the `react-devtools-inline` dependency from 4.4.0 to 4.13.3. A lot of [bug fixes and improvements](https://github.com/facebook/react/blob/master/packages/react-devtools/CHANGELOG.md) have been made since 4.4.0 (released almost a year and a half ago), particularly recently ones related to Fast Refresh that I think might fix [issues users are currently having](https://twitter.com/ggsimm/status/1395035883544453129).

Unfortunately I could not even update the dependency (in `yarn.lock`) because of `node-gyp` failures when Yarn installing (similar to #1846, #4240, #3567).

I'm using OS X 11.3.1 and the latest Node 10 (10.24.1) as specified in the contributing guide. I also tried deleting the `node-gyp` cache folder in my Libraries directory and rebuilding.

@smhutch Sure thing.
```
$ sw_vers
ProductName:	macOS
ProductVersion:	11.3.1
BuildVersion:	20E241

$ node -v
Now using node v10.24.1 (npm v6.14.12)

$ yarn
...
error /Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-pty: Command failed.
Exit code: 1
Command: node scripts/install.js
Arguments:
Directory: /Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-pty
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@5.0.3
gyp info using node@10.24.1 | darwin | x64
gyp info find Python using Python version 2.7.16 found at "/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python"
gyp info spawn /System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python
gyp info spawn args [ '/Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-pty/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/Users/bvaughn/Library/Caches/node-gyp/10.24.1/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/Users/bvaughn/Library/Caches/node-gyp/10.24.1',
gyp info spawn args   '-Dnode_gyp_dir=/Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/Users/bvaughn/Library/Caches/node-gyp/10.24.1/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-pty',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.' ]
gyp info spawn make
gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
In file included from ../src/unix/pty.cc:20:
In file included from ../../nan/nan.h:55:
In file included from /Users/bvaughn/Library/Caches/node-gyp/10.24.1/include/node/uv.h:66:
In file included from /Users/bvaughn/Library/Caches/node-gyp/10.24.1/include/node/uv/unix.h:28:
In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/dirent.h:135:
/Applications/Xcode_11.4.0_fb.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdlib.h:110:82: error: use of undeclared identifier 'labs'; did you mean 'abs'?
inline _LIBCPP_INLINE_VISIBILITY long      abs(     long __x) _NOEXCEPT {return  labs(__x);}
                                                                                 ^
/Applications/Xcode_11.4.0_fb.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdlib.h:110:44: note: 'abs' declared here
inline _LIBCPP_INLINE_VISIBILITY long      abs(     long __x) _NOEXCEPT {return  labs(__x);}
                                           ^
/Applications/Xcode_11.4.0_fb.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdlib.h:112:81: error: use of undeclared identifier 'llabs'
inline _LIBCPP_INLINE_VISIBILITY long long abs(long long __x) _NOEXCEPT {return llabs(__x);}
                                                                                ^
/Applications/Xcode_11.4.0_fb.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdlib.h:115:35: error: unknown type name 'ldiv_t'; did you mean 'dev_t'?
inline _LIBCPP_INLINE_VISIBILITY  ldiv_t div(     long __x,      long __y) _NOEXCEPT {return  ldiv(__x, __y);}
                                  ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/sys/_types/_dev_t.h:31:31: note: 'dev_t' declared here
typedef __darwin_dev_t        dev_t;    /* device number */
                              ^
In file included from ../src/unix/pty.cc:20:
In file included from ../../nan/nan.h:55:
In file included from /Users/bvaughn/Library/Caches/node-gyp/10.24.1/include/node/uv.h:66:
In file included from /Users/bvaughn/Library/Caches/node-gyp/10.24.1/include/node/uv/unix.h:28:
In file included from /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/dirent.h:135:
/Applications/Xcode_11.4.0_fb.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdlib.h:115:95: error: use of undeclared identifier 'ldiv'; did you mean 'div'?
inline _LIBCPP_INLINE_VISIBILITY  ldiv_t div(     long __x,      long __y) _NOEXCEPT {return  ldiv(__x, __y);}
                                                                                              ^
/Applications/Xcode_11.4.0_fb.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdlib.h:115:42: note: 'div' declared here
inline _LIBCPP_INLINE_VISIBILITY  ldiv_t div(     long __x,      long __y) _NOEXCEPT {return  ldiv(__x, __y);}
                                         ^
/Applications/Xcode_11.4.0_fb.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdlib.h:117:34: error: unknown type name 'lldiv_t'
inline _LIBCPP_INLINE_VISIBILITY lldiv_t div(long long __x, long long __y) _NOEXCEPT {return lldiv(__x, __y);}
                                 ^
/Applications/Xcode_11.4.0_fb.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/stdlib.h:117:94: error: use of undeclared identifier 'lldiv'
inline _LIBCPP_INLINE_VISIBILITY lldiv_t div(long long __x, long long __y) _NOEXCEPT {return lldiv(__x, __y);}
                                                                                             ^
In file included from ../src/unix/pty.cc:20:
In file included from ../../nan/nan.h:55:
In file included from /Users/bvaughn/Library/Caches/node-gyp/10.24.1/include/node/uv.h:66:
In file included from /Users/bvaughn/Library/Caches/node-gyp/10.24.1/include/node/uv/unix.h:31:
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:538:2: error: unknown type name 'uint32_t'
        uint32_t                gr_interface;   /* interface index */
        ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:543:2: error: unknown type name 'uint32_t'
        uint32_t                gsr_interface;  /* interface index */
        ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:556:2: error: unknown type name 'uint32_t'
        uint32_t                 msfr_ifindex;  /* interface index */
        ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:557:2: error: unknown type name 'uint32_t'
        uint32_t                 msfr_fmode;    /* filter mode for group */
        ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:558:2: error: unknown type name 'uint32_t'
        uint32_t                 msfr_nsrcs;    /* # of sources in msfr_srcs */
        ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:559:2: error: unknown type name 'uint32_t'
        uint32_t                __msfr_align;
        ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:574:66: error: unknown type name 'uint32_t'
int     setipv4sourcefilter(int, struct in_addr, struct in_addr, uint32_t,
                                                                 ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:575:5: error: unknown type name 'uint32_t'
    uint32_t, struct in_addr *) __OSX_AVAILABLE_STARTING(__MAC_10_7, __IPHONE_4_3);
    ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:576:66: error: unknown type name 'uint32_t'
int     getipv4sourcefilter(int, struct in_addr, struct in_addr, uint32_t *,
                                                                 ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:577:5: error: unknown type name 'uint32_t'
    uint32_t *, struct in_addr *) __OSX_AVAILABLE_STARTING(__MAC_10_7, __IPHONE_4_3);
    ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:578:30: error: unknown type name 'uint32_t'
int     setsourcefilter(int, uint32_t, struct sockaddr *, socklen_t,
                             ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:579:5: error: unknown type name 'uint32_t'
    uint32_t, uint32_t, struct sockaddr_storage *) __OSX_AVAILABLE_STARTING(__MAC_10_7, __IPHONE_4_3);
    ^
/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/netinet/in.h:579:15: error: unknown type name 'uint32_t'
    uint32_t, uint32_t, struct sockaddr_storage *) __OSX_AVAILABLE_STARTING(__MAC_10_7, __IPHONE_4_3);
              ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
20 errors generated.
make: *** [Release/obj.target/pty/src/unix/pty.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-gyp/lib/build.js:196:23)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Darwin 20.4.0
gyp ERR! command "/Users/bvaughn/.nvm/versions/node/v10.24.1/bin/node" "/Users/bvaughn/Documents/git/codesandbox-client/node_modules/.bin/node-gyp" "rebuild"
gyp ERR! cwd /Users/bvaughn/Documents/git/codesandbox-client/node_modules/node-pty
gyp ERR! node -v v10.24.1
```

--- a/packages/app/package.json
+++ b/packages/app/package.json
@@ -349,7 +349,6 @@
     "sass-loader": "^7.1.0",
     "script-ext-html-webpack-plugin": "^2.1.4",
     "selenium-webdriver": "^4.0.0-alpha.1",
-    "stmux": "^1.8.0",
     "strip-ansi": "3.0.1",
     "style-loader": "^0.21.0",
     "svgo": "^1.3.0",

